homeassistant/components/ruckus_unleashed/config_flow.py

```diff
@@ -28,34 +41,38 @@ async def validate_input(hass: core.HomeAssistant, data):
     """
 
     try:
-        ruckus = await Ruckus.create(
+        async with AjaxSession.async_create(
```

Comment:
Please use the helper function to get the ClientSession

Reply:
In order to support Ruckus ZoneDirector and older Ruckus Unleashed installs I needed ```set_ciphers("DEFAULT")``` on the ClientSession's internal SSLContext. Your helper doesn't support passing in a custom context & the one it uses internally inside ```get_default_no_verify_context()``` doesn't do this.